internal/alb/ls/rules.go

```diff
@@ -305,6 +305,9 @@ func rulesChangeSets(current, desired []elbv2.Rule) (add []elbv2.Rule, modify []
 func condition(field string, values ...string) *elbv2.RuleCondition {
 	return &elbv2.RuleCondition{
 		Field:  aws.String(field),
+		PathPatternConfig: *elbv2.PathPatternConditionConfig{
```

Comment:
```suggestion
		PathPatternConfig: &elbv2.PathPatternConditionConfig{
```

looking at the logs from ci https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_aws-alb-ingress-controller/1093/pull-aws-alb-ingress-controller-e2e-test/1202761243954253825#1:build-log.txt%3A244

```
internal/alb/ls/rules.go:308:22: invalid indirect of elbv2.PathPatternConditionConfig literal (type elbv2.PathPatternConditionConfig)
```